### PR TITLE
Deletes PPO notebook and provides a replacement. PPO hyper-parameter doc tweaks .

### PR DESCRIPTION
**Pull request by eshvk**

### CODE REVIEW

File: docs/Getting-Started-with-Balance-Ball.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ each element of the vector means is defined by the agent logic (the PPO
 training process just learns what values are better given particular state 
 observations based on the rewards received when it tries different values). 
 For example, an element might represent a force or torque applied to a 
-Rigidbody in the agent. The **Discrete** action vector space defines its
+Rigid body in the agent. The **Discrete** action vector space defines its
```
Comment (mmattar) [inline docs/Getting-Started-with-Balance-Ball.md]: I thought this was referencing the physics Rigidbody class. So maybe either "rigid body" or Rigidbody and reference https://docs.unity3d.com/Manual/class-Rigidbody.html

File: docs/Getting-Started-with-Balance-Ball.md [hunk 2]
```diff
@@ -249,25 +249,37 @@ example algorithm for use with ML-Agents. For more information on PPO,
...
 
-1. Open `python/PPO.ipynb` notebook from Jupyter.
-2. Set `env_name` to the name of your environment file earlier.
-3. (optional) In order to get the best results quickly, set `max_steps` to 
-50000, set `buffer_size` to 5000, and set `batch_size` to 512.  For this 
-exercise, this will train the model in approximately ~5-10 minutes.
-4. (optional) Set `run_path` directory to your choice. When using TensorBoard 
-to observe the training statistics, it helps to set this to a sequential value 
+To train the agents within the Ball Balance environment, we will be using the python 
+package. We have provided a convenient python wrapper script called `learn.py` which can be passed
```
Comment (mmattar) [inline docs/Getting-Started-with-Balance-Ball.md]: "which can be passed arguments which are used to configure the training" --> "which accepts arguments used to configure the training phase"

File: docs/Getting-Started-with-Balance-Ball.md [hunk 3]
```diff
@@ -249,25 +249,37 @@ example algorithm for use with ML-Agents. For more information on PPO,
...
+
+To summarize, go to your command line, enter the `ml-agents` directory and type: 
+
+```
+
+python python/learn.py <env_file_path> --run-id=<run-identifier> --train 
+
+```
+
+The `--train` flag tells ML Agents to run in training mode. `env_file_path` should be the path to the Unity executable
```
Comment (mmattar) [inline docs/Getting-Started-with-Balance-Ball.md]: ML-Agents

File: docs/Getting-Started-with-Balance-Ball.md [hunk 4]
```diff
@@ -249,25 +249,37 @@ example algorithm for use with ML-Agents. For more information on PPO,
...
+that was just created. 
+
 
 ### Observing Training Progress
-In order to observe the training process in more detail, you can use 
-TensorBoard. In your command line, enter into `python` directory and then run :
+
+Once you start training using `learn.py` in the way described in the previous section, the `ml-agents` folder will 
+contain a `summaries` directory. In order to observe the training process 
+in more detail, you can use  TensorBoard. From the command line run :
```
Comment (mmattar) [inline docs/Getting-Started-with-Balance-Ball.md]: remove extra space between "use" and "TensorBoard"

File: docs/Getting-Started-with-Balance-Ball.md [hunk 5]
```diff
@@ -329,9 +341,10 @@ under `Assets` -> `ML-Agents` -> `Plugins` -> `Computer`
 
 ### Embedding the trained model into Unity
 
-1. Run the final cell of the notebook under "Export the trained TensorFlow
-graph" to produce an `<env_name >.bytes` file.
-2. Move `<env_name>.bytes` from `python/models/ppo/` into 
+1. The trained model is stored in `models/<run-identifier` in the `ml-agents` folder. Once the
```
Comment (mmattar) [inline docs/Getting-Started-with-Balance-Ball.md]: add missing ">"

File: docs/Getting-Started-with-Balance-Ball.md (hunk 1 above)
Comment (awjuliani) [reply docs/Getting-Started-with-Balance-Ball.md]: Yeah. The Unity rigidbody is always treated as a single world.

File: docs/Training-PPO.md [hunk 6]
```diff
@@ -73,10 +56,23 @@ Typical Range: `32` - `2048`
 
 #### Max Steps
 
-`max_steps` corresponds to how many steps of the simulation (multiplied by frame-skip) are run durring the training process. This value should be increased for more complex problems.
+`max_steps` corresponds to how many steps of the simulation (multiplied by frame-skip) are run during the training process. This value should be increased for more complex problems.
 
 Typical Range: `5e5 - 1e7`
 
+#### Beta (Used only in Discrete Control)
```
Comment (awjuliani) [inline docs/Training-PPO.md]: Newer research (see https://arxiv.org/abs/1801.01290 for example) shows that entropy bonuses are good for continuous control too, so it now applies to both actually.

File: docs/Getting-Started-with-Balance-Ball.md (hunk 1 above)
Comment (eshvk) [reply docs/Getting-Started-with-Balance-Ball.md]: Good catch. I thought we didn't mean the word in a CS sense.  Convention should be `ClassName` if we are referencing a data structure.

**Comment by eshvk** [comment]
@awjuliani @mmattar Please check updated commit

File: docs/Training-PPO.md [hunk 7]
```diff
@@ -73,10 +56,23 @@ Typical Range: `32` - `2048`
 
 #### Max Steps
 
-`max_steps` corresponds to how many steps of the simulation (multiplied by frame-skip) are run durring the training process. This value should be increased for more complex problems.
+`max_steps` corresponds to how many steps of the simulation (multiplied by frame-skip) are run during the training process. This value should be increased for more complex problems.
 
 Typical Range: `5e5 - 1e7`
```
Comment (vincentpierre) [inline docs/Training-PPO.md]: To be consistent : `5e5` - `1e7` instead of `5e5 - 1e7`

**Pull request merged**